sweater/src/test/java/com/pavlov/sweater/domain/RoleTest.java

```diff
@@ -0,0 +1,16 @@
+package com.pavlov.sweater.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RoleTest {
+    @Test
+    void test() {
+        Role role = Role.USER;
+        assertEquals("USER", role.getAuthority());
```

Comment:
I think you can try Role.USER.getAuthority() instead of using a simple string "USER", in case of changing the string value in enum